src/olympia/reviewers/tests/test_forms.py

```diff
@@ -140,7 +145,112 @@ def test_canned_responses(self):
         assert self.cr_theme.response in choices[0]
         assert len(choices) == 1  # No addon response
 
+    def test_reasons(self):
+        self.reason_1 = ReviewActionReason.objects.create(
+            name='reason 1',
+            is_active=True,
+        )
+        self.reason_2 = ReviewActionReason.objects.create(
+            name='reason 2',
+            is_active=True,
+        )
+        self.inactive_reason = ReviewActionReason.objects.create(
+            name='inactive reason',
+            is_active=False,
+        )
+        form = self.get_form()
+        choices = form.fields['reasons'].choices
+        assert len(choices) == 2  # Only active reasons
+        assert list(choices.queryset)[0] == self.reason_1
+        assert list(choices.queryset)[1] == self.reason_2
```

Comment:
We don't do anything specific for the ordering, so it's going to order by `created` (the default for our models), which is fine, but in tests it can be a little flaky if two instances are created in the same second, as we don't store with better precision than that (that makes the ordering undefined).

Couple ways to work around that usually:
- Update one of the instances so that it has a `created` well in the past (there is a `self.days_ago(x)` helper)
- Ignore the ordering in the test and use sorted() when comparing. Typically when doing that we'd compared only ids

Or you could bite the bullet and add a field to sort on, but that seems overkill at the moment.

Reply:
We might actually want these sorted by `name` anyway. I think having them sorted by `created` for the select list will just make things confusing for users. If I make the default sort `name` in the model, then I can rely on that in the test, correct?

Reply:
Yes, that sounds good